zerver/lib/email_mirror.py

```diff
@@ -14,12 +15,13 @@
 from zerver.lib.upload import upload_message_image
 from zerver.lib.utils import generate_random_token
 from zerver.models import Stream, Recipient, get_user_profile_by_email, \
-    get_user_profile_by_id, get_display_recipient, get_recipient
+    get_user_profile_by_id, get_display_recipient, get_recipient, UserMessage, Realm, UserProfile
 import six
 
 logger = logging.getLogger(__name__)
 
 def redact_stream(error_message):
+    # type:(str) -> str
```

Comment:
There should be a space between the `:` and `(` in these annotations; can you fix that everywhere?